tests/subsys/event_manager/testcase.yaml

```diff
@@ -7,3 +7,13 @@ tests:
       - nrf9160dk_nrf9160_ns
       - qemu_cortex_m3
     tags: event_manager
+  event_manager.size_enabled:
+    extra_args: OVERLAY_CONFIG=overlay-event_size.conf
+    platform_exclude: native_posix qemu_x86
```

Comment:
Do we have to exclude these platforms for some reason now ?

Reply:
Removed